cmd/create.go

```diff
@@ -357,6 +359,24 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 				"imageVersion":   "latest",
 			},
 		}, nil
+	} else if provider == "linode" {
+		// Image:
```

Comment:
Please move this comment into the provider code itself.

Reply:
This is to explain why we use these values for `OS` and `Plan` on line 371 below:
```
		return &provision.BasicHost{
			Name:     name,
			OS:       "linode/ubuntu16.04lts",
			Plan:     "g6-nanode-1",
```

This is just like other providers, we have comments in this create.go explaining how/where do we get OS, Plan, etc..

Reply:
OK, no problem.